lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -246,12 +254,72 @@ class PatternStopContents extends Component<Props, State> {
     updatePatternStops(activePattern, patternStops)
   }
 
+  _onPickupChange = (evt: SyntheticInputEvent<HTMLInputElement>) => {
```

Comment:
It almost reads like this function is repeated 4 times... I recommend refactoring.
Maybe we only need one of these methods for all 4 selectors
if we can put the field name as the selector name (`e.target.name`).